excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/MissingUserRolesException.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Exception thrown when user roles defined in Case Definition file are missing.
+ */
+public class MissingUserRolesException extends RuntimeException {
+
+    public MissingUserRolesException(Set<String> missingUserRoles) {
```

Comment:
Does this exception appear in the logs as an error with a stack trace? If yes I'd say to avoid that so we don't clutter the logs. It's enough to log missing roles as warn

Reply:
I don't think it will appear in logs.. I will confirm on this